Services/CO.CDP.Organisation.WebApi/UseCase/UpdateOrganisationUseCase.cs

```diff
@@ -97,24 +97,36 @@ public async Task<bool> Execute((Guid organisationId, UpdateOrganisation updateO
 
                 break;
             case OrganisationUpdateType.AdditionalIdentifiers:
-                if (updateObject.AdditionalIdentifiers == null)
+                if (updateObject.AdditionalIdentifiers == null || !updateObject.AdditionalIdentifiers.Any())
                 {
                     throw new InvalidUpdateOrganisationCommand("Missing additional identifiers.");
                 }
 
                 foreach (var identifier in updateObject.AdditionalIdentifiers)
                 {
+                    if (string.IsNullOrWhiteSpace(identifier.Id))
+                    {
+                        throw new InvalidUpdateOrganisationCommand($"Missing Identifier Number for scheme '{identifier.Scheme}'.");
+                    }
+
+                    // Check if Identifier number already exists
```

Comment:
Maybe we don't need so many comments

Reply:
AdditionalIdentifier part has become complicated so to for better reading needed to add.